docs/source/adodb.asciidoc

```diff
@@ -20,6 +20,7 @@ type = "adodb"
 connection_string = "<ADODB connection_string>"
 connection_string_path = "<path to connection string>"
 query_string_parameters = false
+connection_timeout_seconds = 0
```

Comment:
I would move this one line up, in order to have it next to the other connection parameters.